src/ui/table/LogListWidget.cpp

```diff
@@ -0,0 +1,111 @@
+#include "LogListWidget.hpp"
+
+#include <QListWidgetItem>
+
+LogListWidget::LogListWidget(QWidget *parent)
+    : QListWidget(parent)
+{
+    setFixedWidth(250);
+}
+
+
+void
+LogListWidget::logCharacterStatChanged(int charRow, int charColumn, bool isAlly)
+{
+    QString listEntry;
+
+    switch (charColumn) {
+    case 0:
+        listEntry = "Changed name ";
+        break;
+    case 1:
+        listEntry = "Set INI ";
+        break;
+    case 2:
+        listEntry = "Set INI modifier ";
+        break;
+    case 3:
+        listEntry = "Set HP ";
+        break;
+    case 5:
+        listEntry = "Added more info ";
+        break;
+    default:
+        const auto appendixString = isAlly ? "'Ally'." : "'Enemy'.";
+        addNewEntry("Set enemy status in row " + QString::number(charRow) + " to " + appendixString);
+        return;
+    }
+
+    addNewEntry(listEntry += "for row " + QString::number(charRow) + ".");
+}
+
+
+void
+LogListWidget::logChangedCharacterCount(int count, bool added)
+{
+    auto newEntry = added ? "Added " : "Removed ";
+    addNewEntry(newEntry + QString::number(count) + " Character(s).");
+}
+
+
+void
```

Comment:
It is a better idea to use less functions. Maybe refactor all those taking one `int` and also those taking two `int` values

Reply:
Done with e4ab03a82b91e14a4b063392f0cac89cf13ecbfd.